interface/modules/zend_modules/public/index.php

```diff
@@ -18,18 +18,20 @@
 use Laminas\Console\Request as ConsoleRequest;
 
 //fetching controller name and action name from the SOAP request
-$urlArray = explode('/', ($_SERVER['REQUEST_URI'] ?? ''));
+$urlArray = explode('/', $_SERVER['REQUEST_URI']);
```

Comment:
pretty sure this was added to avoid php8 warnings when run zend via commandline (ie. ccda patient import stuff). minimal issues and will confirm this when spend more time on the ccda import stuff in future.

Reply:
good eye. unintentional revert.